src/Microsoft.ML.Data/DataLoadSave/Text/TextLoaderStatic.cs

```diff
@@ -144,6 +144,15 @@ internal Context(Reconciler rec)
             /// <returns>The column representation.</returns>
             public Vector<bool> LoadBool(int minOrdinal, int? maxOrdinal) => Load<bool>(DataKind.BL, minOrdinal, maxOrdinal);
 
+            /// <summary>
+            /// Create a representation for a key loaded from TextLoader as an unsigned integer (32 bits).
+            /// </summary>
+            /// <param name="ordinal">The zero-based index of the field to read from.</param>
+            /// <param name="minKeyValue">smallest value of the loaded key values</param>
+            /// <param name="maxKeyValue">If specified, it's the largest allowed value of the loaded key values. Use null if key is unbounded.</param>
+            /// <returns>The column representation.</returns>
+            public Key<uint> LoadUInt32Key(int ordinal, ulong minKeyValue, ulong? maxKeyValue) => Load<uint>(DataKind.U4, ordinal, minKeyValue, maxKeyValue);
```

Comment:
>LoadUInt32Key [](start = 29, length = 13)

Maybe just `LoadKey` ?

Reply:
Ok.

---
In reply to: [229029785](https://github.com/dotnet/machinelearning/pull/1407#discussion_r229029785) [](ancestors = 229029785)